README.md

```diff
@@ -43,120 +36,74 @@ Check the original technical report
 
 In addition, fields from the following extensions must be imported in the item:
 - [Scientific Extension Specification][stac-ext-sci] to describe relevant publications.
-- [EO Extension Specification][stac-ext-eo] to describe eo data.
 - [Version Extension Specification][stac-ext-ver] to define version tags.
 
 [stac-ext-sci]: https://github.com/radiantearth/stac-spec/tree/v1.0.0-beta.2/extensions/scientific/README.md
-[stac-ext-eo]: https://github.com/radiantearth/stac-spec/tree/v1.0.0-beta.2/extensions/eo/README.md
 [stac-ext-ver]: https://github.com/radiantearth/stac-spec/tree/v1.0.0-beta.2/extensions/version/README.md
 
-### Data Object
 
-| Field Name      | Type                                       | Description                                                                                                                   |
-|-----------------|--------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------|
-| process_level   | [Process Level Enum](#process-level-enum)  | Data processing level that represents the apparent variability of the data.                                                   |
-| data_type       | [Data Type Enum](#data-type-enum)          | Data type (`uint8`, `uint16`, etc.) enum based on numpy base types for data normalization and pre-processing.                 |
-| nodata          | integer \| string                          | Value indicating *nodata*, which could require special data preparation by the network (see [No Data Value](#no-data-value)). |
-| number_of_bands | integer                                    | Number of bands used by the model                                                                                             |
-| useful_bands    | \[[Model Band Object](#model-band-object)] | Describes bands by index in the relevant order for the model input.                                                           |
+### Model Input
 
-#### Process Level Enum
+| Field Name              | Type                            | Description                                                                                                                     |
+|-------------------------|---------------------------------|---------------------------------------------------------------------------------------------------------------------------------|
+| name                    | string                          | Informative name of the input variable. Example "RGB Time Series"                             |
+| bands            | [Band](#bands)                       | Describes the EO data used to train or fine-tune the model.            |
+| input_array           | [Array](#array) | Shape of the input array/tensor ($N \times C \times H \times W$).                                                                     |
```

Comment:
ah thanks for pointing these out, missed updating the description for this field.

I used "Array" since I've noticed outside of the deep learning community the word "Tensor" can be intimidating to folks, and really the term Array is equivalent for how they are used in deep learning. But I see that this could be confused with JSON array types now so agree we should change the term.

What do you think about "NDArray", short for N-Dimensional Array? I'm also considering @PondiB's comment that this extension could be used for non-deep learning models, where it is atypical to refer to model inputs as tensors.

Reply:
NDArray is very strongly related to numpy in the python community. Similarly, I can see Tensor being strongly related to pytorch or tensorflow. Maybe the best would be to have an entirely different name? Something like "Input Structure", since what is important here is more the shape/dtype/axe-order over the actual implementation.

Reply:
Gotcha yeah good points.

I like including "array" in the name, since an array has a shape, dtype, axis ordering. What about "Feature Array Object"? Machine learning models take features as inputs, the features we are looking to support here are arrays. Calling it an Object should help disambiguate with JSON arrays since JSON arrays are not objects.

Reply:
Feature as a similar problem that can be confused with GeoJSON definitions. Since this extension could be combined with other STAC Items resembling GeoJSON feature collections, I would prefer to avoid it.

Reply:
oof ok you're right... 

I'd like to use terminology that is broadly applicable to computer vision and remote sensing communities, something that clearly signals that this extension is for models that primarily take arrays as inputs (rather than tabular data).

Array Object seems to me to be the best compromise that signals clearly what the described input is, as long as we can get past that JSON also specifies an array type. 

Would you be ok with using `Array Object` given that we specify Object, and JSON arrays can't be objects? I can also add a note to disambiguate, saying that this is not a JSON array, it describes a model input array.